nomt/src/cursor.rs

```diff
@@ -302,7 +536,7 @@ impl nomt_core::Cursor for PageCacheCursor {
     }
 
     fn seek(&mut self, path: KeyPath) {
-        PageCacheCursor::seek(self, path)
+        PageCacheCursor::seek(self, path, SeekMode::PathOnly);
```

Comment:
Nit:

```suggestion
        let _ = PageCacheCursor::seek(self, path, SeekMode::PathOnly);
```

I saw this piece but I didn't see the `;` and did not realize that it returns a result now. Explicitly indicating discarding the result would be clearer.